File: PFSC/PFSC.Entities/Entites/PrintingProduction.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace PFSC.Entities.Entites
{
    public class PrintingProduction
    {
        public int Id { get; set; }
        public string Title { get; set; }
        public string Description { get; set; }
        public ICollection<FactoryToProduction> Factories { get; set; }

        public PrintingProduction()
        {
            Factories = new List<FactoryToProduction>();
        }
    }
}

File: PFSC/PFSC.WebUI/ClientApp/app/models/printingProduction.ts
export class PrintingProduction {
    id: number;
    title: string;
    description: string;
    isChecked: boolean;

    constructor(id: number, title:string, desc:string ) {
        this.id = id;
        this.title = title;
        this.description = desc;
    }
}